### DISCUSSION TITLE
A question about environmental variables

### DISCUSSION THREAD

**Discussion by Isaac-Tait** [tinacms/tinacms#4050] [General]
I am curious about this line in my `tina/__generated__/client.js` file:

`export const client = createClient({ url: 'https://content.tinajs.io/1.4/content/<myFullUnobfuscatedClientID>/github/main', token: '<myCompleteUnobfuscatedToken>', queries });`

---

[In the setup docs](https://tina.io/docs/frameworks/nextjs/#step-3-connect-the-tina-cloud-config-to-your-site) it states that `You should apply the token above in an environment variable, instead of hard-coding it.` I am assuming that this is for security purposes. However, when I pushed `tina/__generated__` afterwards I noticed that in the  `client.js` file the clientID and token are not obfuscated with a `.env.VARIABLE` 

The docs state that one should push the **entire** ``__generated__`` contents to GitHub to avoid errors (https://tina.io/docs/errors/faq/#tina-directory-not-pushed-to-git) but so far I have not read in the docs to make sure to apply enviromental variables to all the pertinent files (not just the `tina/__generated__/config.js` file)

**Comment by kldavis4** [comment]
The docs needs to be updated there. 

The `__generated__` folder in `tina` (note that it is not `.tina`) is no longer committed to github. See this [starter](https://github.com/tinacms/tina-cloud-starter/tree/main/tina) for what you should have in github.
